### PR TITLE
[NO-TASK] Fix issues after updating to v.0.9.7

### PR DESCRIPTION
**Pull request by lut1k**
**Description:**

Fixed issues with frontend connection after node update.

**Changes in storage or API:**

--

**Checklist**

- [ ] Code is covered with tests
- [ ] Code is covered with benchmarks
- [ ] Fresh benchmark results included

### CODE REVIEW

**Review by Jekahome — APPROVED** [review]

File: types.json [hunk 1]
```diff
@@ -1,191 +1,202 @@
...
+          "reserved": "Balance",
+          "misc_frozen": "Balance",
+          "fee_frozen": "Balance"
+        }
+      },
+      "AccountData": {
+        "free": "Balance",
+        "reserved": "Balance",
+        "frozen": "Balance"
       }
```
Comment (msifd) [inline types.json:199]: mb add alias types `SubstrateAccountData` and `OrmlAccountData` to remove nested type at 188 and make the purpose of this differentiation a bit clearer?

Comment (lut1k) [reply types.json]: Added alias `SubstraAccountData`. Check, please

Comment (msifd) [reply types.json]: Add alias for orml as well 

     "OrmlAccountData": {
        "free": "Balance",
        "reserved": "Balance",
        "frozen": "Balance"
      },
      "AccountData": "OrmlAccountData",

Comment (lut1k) [reply types.json]: Added

**Review by msifd — APPROVED** [review]

**Review by Apollo50 — APPROVED** [review]

**Pull request merged**